ReactWindows/ReactNative/Views/Text/ReactTextShadowNode.cs

```diff
@@ -117,7 +125,13 @@ public override void OnCollectExtraUpdates(UIViewOperationQueue uiViewOperationQ
 
             if (_inline != null)
             {
-                uiViewOperationQueue.EnqueueUpdateExtraData(ReactTag, _inline);
+                var args = new Tuple<Inline, TextAlignment?, int?, double?>(
+                    _inline,
+                    _textAlignment,
+                    _numberOfLines,
+                    _lineHeight);
```

Comment:
I wonder if setting the extra data to an Action<TextBlock> would be more maintainable, and setting on the properties on the text block here in the shadow node. Thoughts?